packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -63,7 +63,7 @@ const CONTENTFUL_IMAGE_MAX_SIZE = 4000
 
 const isImage = image =>
```

Comment:
I'm not 100% sure removing the ?. is correct here. I can't tell that image is not going to be null. In "packages/gatsby-source-contentful/src/fetch.js" I knew that e was never going to be undefined/null.